packages/block-library/src/comment-template/test/index.js

```diff
@@ -0,0 +1,58 @@
+import { convertToTree } from '../util';
+
+describe( 'Convert to tree', () => {
+	it( 'Empty comments', () => {
+		const comments = convertToTree( [] );
+
+		expect( comments ).toMatchInlineSnapshot( `Array []` );
+	} );
+
+	it( 'Handle comments', () => {
+		const comments = convertToTree( [
+			{ commentId: 1, parent: 0 },
+			{ commentId: 2, parent: 0 },
+			{ commentId: 3, parent: 2 },
+			{ commentId: 4, parent: 2 },
+			{ commentId: 5, parent: 4 },
+			{ commentId: 6, parent: 1 },
+		] );
+
+		expect( comments ).toMatchInlineSnapshot( `
```

Comment:
You're testing a js util here so no need to use `toMatchInlineSnapshot` - this is for `html`. You can just use `toEqual` and the rest like `expect.objectContaining`, etc..

Reply:
got it! I sometimes use `toMatchInlineSnapshot` like a "better console.log" when initially writing the tests 🙂